pkgs/applications/networking/ftp/filezilla/default.nix

```diff
@@ -19,11 +19,11 @@
 
 stdenv.mkDerivation rec {
   pname = "filezilla";
-  version = "3.63.1";
+  version = "3.64.0";
```

Comment:
Would you mind updating to 3.66.4+ so we can get the mitigation for CVE-2023-48795?

Reply:
Done